projects/go-lib/src/lib/components/go-off-canvas/go-off-canvas-submit-button/go-off-canvas-submit-button.component.html

```diff
@@ -0,0 +1,9 @@
+<ng-template goPortalAttachTo="offCanvasSubmitButton">
+  <button
+    class="go-off-canvas__submit-button"
+    type="submit"
```

Comment:
Probably should also just make this a binding and default it to `submit`. I am sure someone, somewhere will want it to be a `button`